### ISSUE TITLE
CustomAvatars crashes Beat Saber while attempting to play a song

### ISSUE THREAD

**Issue by CCF100** [nicoco007/BeatSaberCustomAvatars#119]
**Before doing anything else, please attach your latest log to this issue. It should be located at `Logs\_latest.log` in your Beat Saber installation's folder.**
[_latest.log](https://github.com/nicoco007/BeatSaberCustomAvatars/files/7026434/_latest.log)

**Desktop (please complete the following information):**
- Operating system: Arch Linux
- Headset and controllers: Valve Index

I'm using this avatar: https://modelsaber.com/Avatars/?id=1572761185&pc
(Although I've noticed it crashes even without an avatar set)

I've already confirmed that Beat Saber works just fine with CustomAvatars uninstalled
I'm using Beat Saber 1.16.4

**Describe the bug**
A clear and concise description of what the bug is. If your issue occurs with a particular avatar, please add a link to where you found it or attach it; **please do not attach the avatar file to this issue if you did not create it**.

**Steps to reproduce**
1. Attempt to play any song (Stock or Custom) with CustomAvatars installed

There's a chance it might be hardware specific, my laptop which has a GTX 1660 Ti M consistently crashes, but my desktop, with its Radeon RX 5600 XT is fine... (Both systems run Arch)

**Expected behavior**
The game working normally and not crashing 🙃

**Comment by robinxan** [comment]
I had the same Problem with my RTX 2080 on Manjaro starting with Version `5.2.6`, back then i solved it by not updating.
With beatsaber `1.16.3` i took a look at the problem.   
The crash is caused by the gameengine/driver not being able to instance a cubemap, this is likely related to the nvidia-driver on linux.

Going back in the commit history i found the offending change in commit 73fb012a8f0755b748c2e1dffed2b31ce7a53d20.
Changing `kBlackCubemap` in `GameEnvironmentObjectManager` from `TextureFormat.DXT1` back to `TextureFormat.DXT1Chrunched` solved it for me.

```diff
diff --git a/Source/CustomAvatar/Player/GameEnvironmentObjectManager.cs b/Source/CustomAvatar/Player/GameEnvironmentObjectManager.cs
index ea8285c..0d920a9 100644
--- a/Source/CustomAvatar/Player/GameEnvironmentObjectManager.cs
+++ b/Source/CustomAvatar/Player/GameEnvironmentObjectManager.cs
@@ -29,7 +29,7 @@ namespace CustomAvatar.Player
     {
         private static readonly int kReflectionProbeTexture1PropertyId = Shader.PropertyToID("_ReflectionProbeTexture1");
         private static readonly int kReflectionProbeTexture2PropertyId = Shader.PropertyToID("_ReflectionProbeTexture2");
-        private static readonly Cubemap kBlackCubemap = new Cubemap(0, TextureFormat.DXT1, false);
+        private static readonly Cubemap kBlackCubemap = new Cubemap(0, TextureFormat.DXT1Crunched, false);

         private readonly DiContainer _container;
         private readonly ILogger<GameEnvironmentObjectManager> _logger;
```

**Comment by CCF100** [comment]
> @robinxan @ChrisJAllan Thank you so much for investigating! I had changed it to `DXT1` thinking it would be better supported, but looks like it's the other way around.
> 
> @CCF100 Here's a build with the fix: [BeatSaberCustomAvatars-0268fa3-RELEASE.zip](https://github.com/nicoco007/BeatSaberCustomAvatars/files/7069707/BeatSaberCustomAvatars-0268fa3-RELEASE.zip)

Thank you!
